test/contour-grid.js

```diff
@@ -1,77 +0,0 @@
-/**
```

Comment:
The beauty of GitHub is that I can delete this things and they will still exist 😉 

These are great debugging tools but they made the example and test folder unwieldy. If they are truly useful I would suggest packaging them up as their own modules.